### PR TITLE
Small changes in README.md

### PR DESCRIPTION
**Pull request by ApurvaH**
Small changes in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +56,7 @@ It requires a Python 2.7 environment already installed.
 
 
 ## Liota.conf
-Right now there is only one item in the liota.conf, where to find a file called logging.json which holds the
-dafault initialization parameters for logging. When initialing, liota looks in the current
-working directory, '.', the user's home directory '~', a LIOTA_CONF environment variable, and
-finally the default location for every install, /etc/liota/conf for liota.conf.
+Right now there is only one item in the liota.conf, where to find a file called logging.json which holds the dafault initialization parameters for logging. When initializing, liota looks in the current working directory, '.', the user's home directory '~', a LIOTA_CONF environment variable and finally the default location for every installation: /etc/liota/conf for liota.conf.
 
 Here is the default, v0.7, liota.conf file
```
Comment (pmasrani) [inline README.md]: Change this to the current version, 0.2 and also change the below config file to the latest one.

File: packages/edge_systems/README.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-Put system (gateway) packages in individual folders under this folder. However, 
-all system packages should have the same name `system.py` (or `gateway.py`).
+Put system (gateway) packages in individual folders under this folder. However, all system packages should have the same name `system.py` (or `gateway.py`).
 
-Refer to Charles' fork to figure out how to keep different system packages for
-different systems.
+Refer to Charles' fork to figure out how to keep different system packages for different systems.
```
Comment (pmasrani) [inline packages/edge_systems/README.md]: Remove this line.

File: README.md [hunk 3]
```diff
@@ -64,10 +56,7 @@ It requires a Python 2.7 environment already installed.
 
 
 ## Liota.conf
-Right now there is only one item in the liota.conf, where to find a file called logging.json which holds the
-dafault initialization parameters for logging. When initialing, liota looks in the current
-working directory, '.', the user's home directory '~', a LIOTA_CONF environment variable, and
-finally the default location for every install, /etc/liota/conf for liota.conf.
+Right now there is only one item in the liota.conf, where to find a file called logging.json which holds the dafault initialization parameters for logging. When initializing, liota looks in the current working directory, '.', the user's home directory '~', a LIOTA_CONF environment variable and finally the default location for every installation: /etc/liota/conf for liota.conf.
```
Comment (KohliDev) [inline README.md]: Use bullet points to explain different directories.

File: packages/README.md [hunk 4]
```diff
@@ -11,23 +11,23 @@ Currently, supported commands include package action commands and statistical co
...
 
-Liota packages must follow certain formats for package manager to process it correctly. It is up to the package developer to follow the format requirements. Details will be provided in later parts of this documents. Please also refer to `packages` and `packages/example` for example packages we provide.
+Liota packages must follow certain formats for package manager to process them correctly. It is up to the package developer to follow the format requirements. Details will be provided in later parts of this document. Please also refer to `packages` and `packages/example` for example packages we have provided.
 
-If dependency lists of specified packages and their denpendencies contain loops, all packages involved in the loop are not to be loaded.
+If dependency lists of specified packages and their dependencies contain loops, all packages involved in the loop are not to be loaded.
 
 **unload** package_name [package_name] ...
```
Comment (KohliDev) [inline packages/README.md]: Use Bullet points.

File: packages/README.md [hunk 5]
```diff
@@ -37,17 +37,17 @@ Remove a package with the specified name. By default, the removed package will b
 
 **stat** met|col|th
```
Comment (KohliDev) [inline packages/README.md]: Same as above use bullet points.

File: README.md (hunk 1 above)
Comment (KohliDev) [inline README.md]: Remove the version number and provide link to logging.json and display all the contents of latest liota.conf.

**Comment by KohliDev** [comment]
Ship It

**Comment by KohliDev** [comment]
Ship It

**Pull request merged**